File: Kattis/Java/Aaah.java
package aaah;

import java.util.Scanner;

public class Aaah {

    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);
        String jonsAaah = input.nextLine();
        String docsAaah = input.nextLine();
        
        if (jonsAaah.length() >= docsAaah.length()) {
            System.out.println("go");
        } else {
            System.out.println("no");
        }
    }
    
}

File: Kattis/CSharp/Aaah.cs
using System;

namespace Aaah_
{
    class Aaah
    {
        static void Main(string[] args)
        {
            string jonsAaah = Console.ReadLine();
            string docsAaah = Console.ReadLine();

            if (jonsAaah.Length >= docsAaah.Length)
                Console.WriteLine("go");
            else 
                Console.WriteLine("no");
        }
    }
}
